node_agent/main.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017 Istio Authors
```

Comment:
Totally! Need to to block on this as we have an issue tracking. Feel free to merge after someone signs off